package.json

```diff
@@ -17,10 +17,12 @@
     "ssl:generate": "node ./certs/devcert.js",
     "heroku-postbuild": "turbo run build --filter api",
     "start": "cd apps/api && npm run start",
-    "ngrok": "ngrok http https://api.sweetr.local:8000 --domain weekly-flying-primate.ngrok-free.app",
-    "test": "vitest run"
+    "test": "vitest run",
+    "_ngrok": "ngrok http 8000 --domain $NGROK_DOMAIN",
+    "ngrok": "dotenv -e ./apps/api/.env -- npm run _ngrok"
   },
   "dependencies": {
+    "@tabler/icons-react": "^3.19.0",
```

Comment:
A couple things:
- I think you added tabler icons here by mistake?
- Didn't we need `dotenv-cli`? Isn't that package the one providing the `dotenv` binary used by the `ngrok` command?